.github/workflows/release.yml

```diff
@@ -122,7 +122,8 @@ jobs:
           name: build
           path: build
 
-      - name: "📂 download diagram artifacts"
+      - if: jobs.build.outputs.build-artifact
```

Comment:
@bdougie not sure I'm doing this right

Reply:
At first look this is fine. I suppose there is one way to test it.

Reply:
I can't merge otherwise I would have already 🔥